pom.xml

```diff
@@ -210,6 +209,16 @@
             </execution>
           </executions>
         </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-release-plugin</artifactId>
+          <version>3.0.1</version>
+          <configuration>
+            <autoVersionSubmodules>false</autoVersionSubmodules>
+            <updateDependencies>false</updateDependencies>  <!-- Verhindert, dass Parent die Submodule ändert -->
```

Comment:
See comments before: If a submodule changes, it needs to get a new version number even if we release all modules via the parent.